code/game/objects/structures/petrified_statue.dm

```diff
@@ -80,12 +80,16 @@
 		return FALSE
 	var/obj/structure/statue/petrified/S = new(loc, src, statue_timer)
 	S.name = "statue of [name]"
-	bleedsuppress = 1
```

Comment:
Where else is `bleedsuppress` used? You should probably replace the other instances with the trait.

Reply:
Ah, thanks for reminding me. Updated it to totally replace it. It was used in highlander, blood trails, and handle_blood.